__test__/ExperimentTable.test.js

```diff
@@ -16,7 +16,8 @@ describe(`ExperimentTable`, () => {
     resource: `bool`,
     enableDownload: true,
     enableIndex: true,
-    TableCellDiv: TableCellDiv
+    TableCellDiv: TableCellDiv,
+    downloadTooltip: `tooth`
```

Comment:
There are two problems here:
1. The test doesn’t convey the fact that the tooltip content can be HTML (and is rendered as such).
2. Just the word `tooth` is too cryptic. If you fell like a dental comment is in order, be a bit more explicit (e.g. `I can’t think of a good tooltip text here, but I can say that my <b>tooth hurts a lot</b>!). Be conscious of future Atlas developers who might not be able to guess why certain hard-coded values were picked!